tiledb/sm/global_state/libcurl_state.cc

```diff
@@ -50,6 +52,11 @@ Status init_libcurl() {
         std::to_string(rc)));
 #endif
 
+#ifdef __linux__
+  VFS vfs;
+  vfs.init(config.vfs_params());
+  constants::cert_file = utils::https::find_ca_certs_linux(vfs);
```

Comment:
Not really a constant if it's being set at runtime. I think the places where this is needed should call `find_ca_certs_linux` themselves.

Reply:
@tdenniston I decided to try to detect only one time since once the CA cert bundle is found once we can cache it because it should not change locations. If we don't cache the location we would run the find for every creation of S3 class or for every rest request. 

I considered it a constant since it is set once at runtime and not set again and not editable by the user. I agree its not `const` in constant so we can move the location of the global variable or drop caching and lookup for each access as you suggested.